components/gateway/internal/gqlschema/schema.graphql

```diff
@@ -14,6 +14,8 @@ scalar QueryParams # -> map[string][]string
 
 scalar CLOB # TBD
 
+scalar PageCursor
```

Comment:
It should be commented that this is just a string. Because scalar can represent dynamic structure as well